plugins/modules/sns_topic.py

```diff
@@ -61,6 +61,9 @@
       protocol:
         description: Protocol of subscription.
         required: true
+      attributes:
```

Comment:
@canidam  thank you for taking time to work on this. Could you please add  `version_added: 2.0`?

Reply:
@alinabuzachis done